test/tests/performance/pbs_client_nagle_performance.py

```diff
@@ -130,6 +140,37 @@ def test_qdel_nagle_perf(self):
         self.logger.info(
             "qdel performance after setting manager: " + str(qdel_perf2))
 
-        # Check that the two timings are pretty close to each other
-        self.assertTrue((qdel_perf2 - qdel_perf) < 5,
-                        "qdel performance differs too much!")
+    @timeout(600)
+    def test_qsub_perf(self):
+        """
+        Test that qsub performance have improved when run
+        with -f option
+        """
+
+        # Restart server again
+        self.server.restart()
+
+        # Submit a job for 500 times and timeit
+        cmd = os.path.join(
+              self.server.client_conf['PBS_EXEC'],
+              'bin',
+              'qsub -- /bin/true >/dev/null')
+        start_time = timeit.default_timer()
+        for x in range(1, 500):
+            rv = self.du.run_cmd(self.server.hostname, cmd)
+            self.assertTrue(rv['rc'] == 0)
+        end_time1 = timeit.default_timer() - float(start_time)
```

Comment:
This isn't really an end time.  It's the difference between the start and end time.  Consider renaming the variable